---

radii: [0.038]
x_pos: [-1.0]
y_pos: [0.0]
x0: 0.0
y0: 0.0
theta0: 0.0
frequency: 500
x_length: 10
y_length: 10